.github/workflows/build.yml

```diff
@@ -156,6 +156,11 @@ jobs:
         working-directory: ./tooling/Scripts/
         run: ./PackEachExperiment.ps1 -date ${{ env.VERSION_DATE }}${{ env.VERSION_PROPERTY != '' && format(' -postfix {0}', env.VERSION_PROPERTY) || '' }}
         
+      # Push Packages to our DevOps Artifacts Feed
+      - name: Add source
+        if: ${{ github.ref == 'refs/heads/main' }}
+        run: dotnet nuget update source LabsFeed --username dummy --password ${{ secrets.DEVOPS_PACKAGE_PUSH_TOKEN }}
+
```

Comment:
I would suggest adding this step after the next one line 172 (as displayed here), so it's the 'Add source' and then 'Push packages' steps that are related right next to each other?

Reply:
We could also just collapse these into a single step anyway, as there'll always be run together, that may be the easiest, just use the same format as the PR version.